internal/os/os.go

```diff
@@ -1,12 +1,260 @@
 package os
 
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"git.sr.ht/~spc/go-log"
+	"github.com/jakub-dzon/k4e-operator/models"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+)
+
 const (
-	UnknownOsImageId = "unknown"
+	UnknownOsImageId                 = "unknown"
+	GreenbootHealthCheckFileName     = "/etc/greenboot/check/required.d/greenboot-health-check.sh"
+	GreenbooFailFileName             = "/etc/greenboot/red.d/bootfail.sh"
+	StartGracefulReboot              = "StartGracefulReboot"
+	FinishedGracefulReboot           = "FinishedGracefulReboot"
+	TimeoutToGracefulRebootInSeconds = 10
+	EdgeConfFileName                 = "/etc/ostree/remotes.d/edge.conf"
```

Comment:
Were you able to verify this is common for all arch/OS?
also, since used internally, we can reduce the visibility to be unexported.

Reply:
@bardielle any update on this?

Reply:
I found that there is an option for configuring the directory creation by a flag add-remotes-config-dir : https://manpages.debian.org/testing/ostree/ostree.repo-config.5.en.html

but not about the  file name

Reply:
It seems to be set it in the ks file - ```ostreesetup --nogpg --url=http://192.168.122.233/device-worker/repo --osname=rhel --remote=edge --ref=rhel/8/x86_64/edge```

Reply:
After investigating it seems the ostree looks for the remotes in the content of the file so the name is not relevant but the remote key name is the thing that importing to mention (which is edge in our case but might be change by the user)

I opened ticket for that